acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/models/TimeSeries.java

```diff
@@ -38,7 +38,8 @@ public String getUrlName() {
             .collect(toList());
 
         // Sort expected publications with latest nominal dates at the top
-        expectedPublications.sort((left, right) -> -1 * left.getNominalDate().compareTo(right.getNominalDate()));
+        expectedPublications.sort((left, right) -> -1 * left.getNominalPublicationDateDate().asInstant()
```

Comment:
Is the asInstant needed to make this work? If so, should maybe comment to explain why.

Reply:
Indeed, it's not, strictly. I'll implement `NominalPublicationDate.compareTo(NominalPublicationDate)`.

Reply:
Done.